sdk/servicelinker/Azure.ResourceManager.ServiceLinker/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-05-23)
 
-### General New Features
+### Features Added
```

Comment:
Why we need to change this line?

Reply:
I'm not sure if the release check is ok for `General New Features`.
Hmm... let me change back and try.